src/Tribe/Date_Utils.php

```diff
@@ -21,18 +21,50 @@ class Tribe__Events__Date_Utils {
 		const DBTIMEFORMAT          = 'H:i:s';
 		const DBYEARMONTHTIMEFORMAT = 'Y-m';
 
+
+		/**
+		 * Get the datepicker format, that is used to translate the option from the DB to a string
+		 *
+		 * @param  int $translate The db Option from datepickerFormat
+		 * @return string|array            If $translate is not set returns the full array, if not returns the `Y-m-d`
+		 */
+		public static function datepicker_formats( $translate = null ) {
```

Comment:
This function would probably make more sense with the name `datepicker_format` as it only returns a single value.